issm-project-week1/src/components/Card.js

```diff
@@ -0,0 +1,27 @@
+import star from "../images/star.png";
+const Card = (props) => {
```

Comment:
Put all files inside components folder in a folderized form like this
![Screenshot from 2024-02-10 19-52-19](https://github.com/hariskhan636/ISSM-Project-Week1/assets/94175765/31d4b8ec-0831-4764-82b7-309c87254677)